test/ctia/flows/crud_test.clj

```diff
@@ -1,10 +1,9 @@
 (ns ctia.flows.crud-test
-  (:require [ctia.flows.crud :as sut]
-            [ctia.lib.collection :as coll]
-            [clj-momo.test-helpers
-             [core :as mth]]
-            [clj-momo.lib.map :refer [deep-merge-with]]
-            [clojure.test :refer [deftest testing is]]))
+  (:require [clj-momo.lib.map :refer [deep-merge-with]]
+            [clojure.test :refer [deftest testing is]]
+            [ctia.auth.threatgrid :refer [map->Identity]]
+            [ctia.flows.crud :as crud]
```

Comment:
I like the convention of using `sut` (system under test).
In any test ns that you look, you immediately now that it refers to the tested ns.

Reply:
I like it over (`sut` naming convention), because:

- It doesn't make it harder to review the code without seeing the entire ns or jumping to the header
- It simplifies grepping
- Overtime, it naturally, slowly forms a convention for naming aliases (familiar names take over things that are named incidentally or without much thinking)
- `sut/preserve-errors` is not very obvious; you still have to look at the filename and/or ns declaration, `crud/preserve-errors` is more pronounced, especially if you're familiar with the namespace where it's defined
- Sometimes, you are not really testing one system in the same namespace; we shouldn't, but sometimes it's inevitable. - I think SUT, CUT (class under test), AUT (application under test), et al., are the recurrence of traits from OOP practices, and it doesn't make sense to use that in Clojure.

But all these points are just my personal opinion; this isn't the kind of hill I'd chose to die on. The point of that change was exactly that: the invitation for a discussion.

Reply:
I like your arguments :-).
Clojure definitively has a problem, we always need to format code for `grep` :-), but with or without `sut` you still can grep the whole namespace to know where it is used. 
In any case it's not blocking, but most of the code mostly uses `sut`.
Since you prefer a more explicit alias that will be consistent across the project, I recommend to pick more than the suffix `crud` that is used in many namespaces (routes, es, some entities and probably others). `flows.crud` will not be ambiguous.